apps-rendering/src/fixtures/item.ts

```diff
@@ -502,6 +512,22 @@ const explainer: Explainer = {
 	outline: outlineFromItem(fields.body),
 };
 
+const immersive: Standard = {
+	design: ArticleDesign.Standard,
+	...fields,
+	display: ArticleDisplay.Immersive,
```

Comment:
I normally put the spread at the top, but I'm wondering if there's a specific reason not to do so here?

```suggestion
    ...fields,
	design: ArticleDesign.Standard,
	display: ArticleDisplay.Immersive,
```

Reply:
I think the order of the spread matters when you're overriding fields. In this case `design` doesn't exist in `fields`, so we're not overriding it, whereas `display` does, so we are.

There's no particular reason to have `design` above the `fields` spread, it's just consistent with how the other fixtures are written.